### PR TITLE
Updated My Username, twitter and linkedIn profile

### PR DESCRIPTION
**Pull request by JayantGoel001**

### CODE REVIEW

File: src/data/contributors/cormackrum.json [hunk 1]
```diff
@@ -1,10 +0,0 @@
-{
```
Comment (Abhishek-kumar09) [inline src/data/contributors/cormackrum.json:1]: You should not remove other's data, please revert

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

File: src/data/contributors/cormackrum.json (hunk 1 above)
Comment (JayantGoel001) [reply src/data/contributors/cormackrum.json:1]: Hello @Abhishek-kumar09 
Actually I have not removed any other's data, I have just updated my own.
I had that username for a while then i updated to this one.

Comment (JayantGoel001) [reply src/data/contributors/cormackrum.json:1]: #129 See this one.

**Comment by Abhishek-kumar09** [comment]
Oh, that was your file previously, OK, thanks for your PR.

**Pull request merged**